components/blitz/src/ome/formats/OMEROMetadataStoreClient.java

```diff
@@ -1719,8 +1719,7 @@ public int getDefaultGroupLevel() throws ServerError {
             else groupLevel = ImportEvent.GROUP_COLLAB_READ;
         }
         else if (perm.isWorldRead()) {
-            if (perm.isWorldWrite())  groupLevel = ImportEvent.GROUP_PUBLIC;
-            else groupLevel = ImportEvent.GROUP_PUBLIC;
+            groupLevel = ImportEvent.GROUP_PUBLIC;
```

Comment:
yes, I suppose we don't have anything further in the enum for world-writable groups